README.md

```diff
@@ -63,57 +58,7 @@ The only thing you need to install is Docker. All of the other dev dependencies
 
     `iotedgedev monitor`
 
-## Overview
-The **Azure IoT Edge Dev Tool** enables you to do all of the following with simple one-line CLI commands.
-
-1. **Start Container**: 
-
-    `docker run -it -v /var/run/docker.sock:/var/run/docker.sock -v c:/temp/iotedge:/home/iotedge microsoft/iotedgedev`
-
-    This container includes all of the dependencies you need for IoT Edge development, including:
-
-    - Docker
-    - .NET Core SDK
-    - Python
-    - Pip
-    - Azure CLI
-
-    You can also directly install with: `pip install iotedgedev`   
-    
-1. **Create Solution**: Create a new IoT Edge Solution that includes a sample module and all the the required configuration files.
-
-    `iotedgedev solution edgesolution1`
-
-    `cd edgesolution1`
-
-1. **Setup Azure**: Creates or selects your Azure IoT Hub and Edge Device and updates your Environment Variables.
-
-    `iotedgedev azure`
-
-    > This must be run from the root of your solution, so make sure you cd into the `edgesolution1` folder before you run this command.
-
-1. **Build, Push & Deploy**: Build, Push and Deploy modules: 
-
-    `iotedgedev push --deploy`
-    
-    > This will `dotnet build`, `publish`, `docker build, tag and push` and `deploy modules` to your IoT Edge device.
-
-    If your module is not dotnet, then the dotnet build/publish steps will be skipped.
-    
-1. **Setup & Start**: Setup and Start the IoT Edge Runtime: 
-
-    `iotedgedev start`
-
-1. **View Messages**: View Messages Sent from IoT Edge to IoT Hub: 
-
-    `iotedgedev monitor`
-
-1. **View Logs**: View and Save Docker log files: 
-
-    `iotedgedev docker --logs`
-
-1. **Setup Custom Registry**: Use a Custom Container Registry: 
-
-    `iotedgedev docker --setup-registry`
+## Resources
```

Comment:
When looking at the output html file, I feel like we are using the reader hanging at the end of the Quickstart.

Can you come up with a good way to say something like CLICK HERE TO VIEW ALL DOCS, not that exactly, but something like that.

Reply:
which links to the wiki? in that case, think it would make sense to pull out the reference to the wiki at the beginning of the resources section and have it here

Reply:
How do you feel about leaving as-is for now and revisit it later?

Reply:
yeah, will be good to re-read it with some fresher eyes

Reply:
Okay, let's still do the link at the top, comment below and call it good.